### PR TITLE
PP-6615 Beginnings of Sending Metrics

### PR DESCRIPTION
**Pull request by rorymalcolm**
- The beginnings of sending metrics to HG

### CODE REVIEW

File: ci/tasks/send-hg-metric.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+run:
+  path: bash
+  dir: src
+  args:
+    - -eu
+    - |
+        apk add netcat-openbsd
+        echo "Sending metric ((METRIC_NAME))"
+        nc "((HOSTED_GRAPHITE_ACCOUNT_ID)).carbon.hostedgraphite.com" 2003 <<<"((HOSTED_GRAPHITE_API_KEY)).((METRIC_NAME)) ((METRIC_VALUE))"
```
Comment (danworth) [inline ci/tasks/send-hg-metric.yml:26]: Maybe just check this works (I know its pretty straight forward) and then link to the metric from HG in the "How have you tested this" section of the PR description?

Comment (rorymalcolm) [reply ci/tasks/send-hg-metric.yml:26]: Added it to the JIRA ticket

File: ci/pipelines/deploy.yml [hunk 2]
```diff
@@ -1167,7 +1167,12 @@ jobs:
           <<: *cf-creds
           CF_SPACE: smoke-tests
           COMMAND: cf ssh smoke-tests-staging -c /app/bin/smoke-cardp1
-
+        on_success:
+          task: notify hg
+          file: omnibus/ci/tasks/send-hg-metric.yml
+          params: 
+            METRIC_NAME: ci.concourse.hosted-graphite.test
```
Comment (danworth) [inline ci/pipelines/deploy.yml:1174]: Have we thought what the naming convention will be? I assume this is just a place holder :)

File: ci/tasks/send-hg-metric.yml (hunk 1 above)
Comment (danworth) [reply ci/tasks/send-hg-metric.yml:26]: 👍

**Pull request closed without merging**